packages/flutter/lib/src/cupertino/list_tile.dart

```diff
@@ -381,12 +381,18 @@ class _CupertinoListTileState extends State<CupertinoListTile> {
     }
 
     return GestureDetector(
-      onTapDown: (_) => setState(() { _tapped = true; }),
-      onTapCancel: () => setState(() { _tapped = false; }),
+      onTapDown: (_) => setState(() {
```

Comment:
This seems fine, I prefer us to consistently use 3 lines for a setState invocation